Add content to Readme file to support "aks debugging session" (#54)
retaildevcrews/docker101: 05-Debugging-apps-in-AKS/README.md

# logapp   0/6     0            0           16m

# The Ready column tell us that our deployment did not create any pods.
# The Ready column shows 0 out of 6 pods are ready.

# Get more detail information about the Deployment.
kubectl describe deployments <deployment name>

# ...
# Conditions:
#   Type             Status  Reason
#   ----             ------  ------
#   Available        False   MinimumReplicasUnavailable
#   ReplicaFailure   True    FailedCreate
#   Progressing      False   ProgressDeadlineExceeded
# ...

# By inspecting different Condition Status entries we can see multiple errors.
# One of the entries refers to Replica Failure.

# Edit Deployment on the cluster.
kubectl edit deployments <deployment name>

# This will let you edit and apply changes directly on the cluster.

# You can use different editors.
KUBE_EDITOR=nano kubectl edit deployments <deployment name> 

# Scaling up or down deployment 
kubectl scale deployment --replicas=<desired number of replicas> <deployment name>

# ...
# deployment.apps/logapp scaled
# ...

# Scale command will allow you to scale up or down the deployment  

```

## Configmaps

```bash
# Get basic Configmaps information.

> # This will tell us if our Configmaps are running. 

Reviewer: update to something like
```suggestion
# This will list the ConfigMaps
```

kubectl get configmap 

# NAME            DATA   AGE
# logapp-config   1      136m

# The Age column tell us how old is the configuration.

> # Get more detail information about the Deployment.

Reviewer: update to something like
```suggestion
# Get more detail information about the ConfigMap.
```

kubectl describe configmap <configmap name>

# ...
# Data
# ====

> # SLEEP_SECONDS:

Reviewer: ```suggestion
# SLEEP_MS:
```

# ----
# 0
# Events:  <none>

> # By inspecting Data section we see that Slepp_Seconds is set to zero.

Reviewer: ```suggestion
# By inspecting Data section we see that SLEEP_MS is set to zero.
```

# There are no Events listed

> # Edit Deployment on the cluster.

Reviewer: ```suggestion
# Edit ConfigMap on the cluster.
```
